.pre-commit-config.yaml

```diff
@@ -11,6 +11,14 @@ repos:
     entry: mypy
     language: system
     types: [python]
+  - id: black
+    name: black
+    description: "Black: The uncompromising Python code formatter"
+    entry: black --line-length=100
+    language: python
+    require_serial: true
+    types: [python]
+    additional_dependencies: [black==23.1.0]
```

Comment:
Hmm, I think I might rather have it listed in requirements-dev.txt. That way, we'll make sure that the version is automatically bumped by Dependabot (and we make sure that it's the same between pre-commit hook and ci/cd).